MaterialLib/SolidModels/Ehlers-impl.h

```diff
@@ -652,7 +708,11 @@ bool SolidEhlers<DisplacementDim>::computeConstitutiveRelation(
     }
 
     // Update sigma.
-    sigma_final.noalias() = G * sigma;
+    if (_damage_properties)
+        sigma_final = G * sigma * (1 - _state.damage);
```

Comment:
Yes you are right, I am currently working on the implementation of the damage correction to the tangent operator.